edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/EdgeMessage.cs

```diff
@@ -13,6 +14,15 @@ public EdgeMessage(byte[] body, IDictionary<string, string> properties, IDiction
             this.Body = Preconditions.CheckNotNull(body, nameof(body));
             this.Properties = Preconditions.CheckNotNull(properties, nameof(properties));
             this.SystemProperties = Preconditions.CheckNotNull(systemProperties, nameof(systemProperties));
+            this.ProcessedPriority = RouteFactory.DefaultPriority;
```

Comment:
Why the naming ProcessedPriority?

Reply:
so that people don't confuse it with priority as when used during routing.  This property is not considered valid until the message is being processed, but since it's on the very generic and widespread IMessage interface, I don't want people to get the wrong idea and start thinking it can be referenced whenever and whereever.

you know what, now I think about I should prob leave a big fat comment on the interface decl to explain this.